### PR TITLE
Switch policy to use RFC3986-compliant predicate type

### PR DESCRIPTION
**Pull request by jdolitsky**
Fixes: Now that we are attesting vuln scans with cosign 2.0, the policy is invalid since predicate types must now be RFC3986-compliant (start with `https://`)
Related: https://github.com/sigstore/policy-controller/issues/608

### Pre-review Checklist

- [ ] **IMPORTANT: 'image-request' tag has been applied if this PR is adding any images, including new versions or variants**

#### For new image PRs only

If you have an apko.yaml file in this PR you need to follow this checklist, otherwise feel free to remove.

- [ ]  Image is marked experimental or stable as appropriate
- [ ]  The last two minor versions are available
- [ ]  The latest tag points to the newest stable version
- [ ]  There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
- [ ]  The image runs as `nonroot` and GID/UID are set to 65532
  - [ ]  Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
  - [ ]  See above for exceptions to nonroot rule
- [ ]  ENTRYPOINT
  - [ ]  For applications/servers/utilities call main program with no arguments e.g. [redis-server]
  - [ ]  For base images leave empty
  - [ ]  For dev variants set to entrypoint script that falls back to system
- [ ]  CMD:
  - [ ]  For server applications give arguments to start in daemon mode (may be empty)
  - [ ]  For utilities/tooling bring up help e.g. `–help`
  - [ ]  For base images with a shell, call it e.g. [/bin/sh]
- [ ]  Consider where and how the image deviates from popular alternatives. Is there a good reason and is it documented?
- [ ]  Add annotations e.g:

        annotations:
          "org.opencontainers.image.authors": "Chainguard Team https://www.chainguard.dev/"
          "org.opencontainers.image.url": https://edu.chainguard.dev/chainguard/chainguard-images/reference/busybox/ #use the academy site here
          "org.opencontainers.image.source": https://github.com/chainguard-images/images/tree/main/images/bazel #use github here

- [ ]  Check if environment variables are needed e.g. to set data locations
- [ ]  Ensure the image responds to SIGTERM
  - [ ]   `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`
- [ ]  Documentation. Let's make this excellent. Include usage example.
- [ ]  Error logs write to stderr and normal logs to stdout. DO NOT write to file.
- [ ]  Include tests, at the very least a basic smoke test.

### CODE REVIEW

**Review by hectorj2f — APPROVED** [review]

**Review by vaikas — APPROVED** [review]
That's the ticket!

File: policies/02-has-vuln-attestation.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ spec:
           policy:
             type: cue
             data: |
-              predicateType: "cosign.sigstore.dev/attestation/vuln/v1"
+              predicateType: "https://cosign.sigstore.dev/attestation/vuln/v1"
```
Comment (vaikas) [inline policies/02-has-vuln-attestation.yaml:23]: You might want to change the L37 below for future proofing.

**Comment by vaikas** [comment]
Ok, seems like the original error has been fixed, but this one is the one that failed now?
```
/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +668bd6305358c8f9472000b99bacc00e5ce9beac:refs/remotes/pull/325/merge
  Error: fatal: unable to access 'https://github.com/chainguard-images/images/': The requested URL returned error: 429
  The process '/usr/bin/git' failed with exit code 128
  Waiting 16 seconds before trying again
  /usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +668bd6305358c8f9[47](https://github.com/chainguard-images/images/actions/runs/4346621087/jobs/7592919798#step:2:52)2000b99bacc00e5ce9beac:refs/remotes/pull/325/merge
  Error: fatal: unable to access 'https://github.com/chainguard-images/images/': The requested URL returned error: 429
  The process '/usr/bin/git' failed with exit code 128
  Waiting 13 seconds before trying again
  /usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +668bd6305358c8f9472000b99bacc00e5ce9beac:refs/remotes/pull/325/merge
  Error: fatal: unable to access 'https://github.com/chainguard-images/images/': The requested URL returned error: 429
  Error: The process '/usr/bin/git' failed with exit code 128
```

**Comment by jdolitsky** [comment]
yea, thats just github actions spurious errors. re-running

**Review by luhring — APPROVED** [review]

**Pull request merged**